src/main/java/lotto/enumeration/LottoReward.java

```diff
@@ -0,0 +1,35 @@
+package lotto.enumeration;
+
+import java.util.Arrays;
+
+public enum LottoReward {
+
+    NO_REWARD(0, 0),
+    THREE_NUMBERS_MATCHED_REWARD(3, 5000),
+    FOUR_NUMBERS_MATCHED_REWARD(4, 50000),
+    FIVE_NUMBERS_MATCHED_REWARD(5, 1500000),
+    SIX_NUMBERS_MATCHED_REWARD(6, 2000000000);
```

Comment:
```suggestion
    SIX_NUMBERS_MATCHED_REWARD(6, 2_000_000_000);
```
언더 스코어를 활용하면 조금 더 가독성을 확보할 수 있습니다 ㅎㅎ

Reply:
이런방식이 있었군요:) 좋은걸 배웠습니다. 감사합니다.